### PR TITLE
annual reports: clarifying end process

### PR DESCRIPTION
**Pull request by parispittman**
drew out the end of annual reports process with some clarified steps and added the issue piece since we set those.

### CODE REVIEW

**Comment by mrbobbytables** [comment]
/lgtm
/hold
for other steering review

File: committee-steering/governance/annual-reports.md [hunk 1]
```diff
@@ -30,9 +30,14 @@ Committee.
...
   - share with their groups (see tips below) and begin to 
-compile the doc due at the end of February 
+compile the doc due April 8th - final version
   - use this [question template]
-  - submit a pull request to the groups documentation in the community repo
+  - submit a pull request with at least the template questions against the set 
+  issue to the groups documentation in the community repo by March 8th
+  - verify that Steering Liaison has reviewed
+  - post a call for comments period with your groups mailing list and a complete 
+  questionairre; you can follow your charter communication period if you have
```
Comment (liggitt) [inline committee-steering/governance/annual-reports.md:39]: questionnaire

**Comment by liggitt** [comment]
one typo, lgtm otherwise

**Pull request merged**